pkg/servicerevision/plan.go

```diff
@@ -0,0 +1,110 @@
+package servicerevision
```

Comment:
Can you explain purporse of the `refactor: deployer and modify revision status` commit? Better to know in commit message